User/pagoe/jun sdk release (#739)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/releasenotes.md

#### Win32 C/C++
* Added [CoreWebView2NewWindowRequestedEventArgs.WindowFeatures]() which fires when window.open() is called. ([#70](https://github.com/MicrosoftEdge/WebViewFeedback/issues/70 ))
* **Breaking Change:** [CreateCoreWebView2EnvironmentWithDetails]() has been deprecated and replaced with [CreateCoreWebView2EnvironmentWithOptions]()
* **Breaking Change:** In order to ensure that our API aligns with the Windows API naming conventions, we have updated the names of the following:
  * [AreRemoteObjectsAllowed]() is now [AreHostObjectsAllowed]()
* Updated [AddRemoteObject]() to ensure original host object serializer markers are set to the proxy objects and serialized back as a host object when passed as a parameter in the JavaScript callback. ([#148](https://github.com/MicrosoftEdge/WebViewFeedback/issues/148))

#### .NET
* Released WinForms and WPF WebView2API Samples, which are comprehensive guides of our SDK. Check out the [WebView2 Samples Repo](https://github.com/MicrosoftEdge/WebView2Samples).
* Added support for visual hosting and window features [experimental APIs](./concepts/versioning.md#experimental-apis)

> * **Breaking Change:** The following deferrals now implement IDisposable: [ScriptDialogOpening](), [NewWindowRequested](), [WebResourceRequested](), and [PermissionRequested]().

Reviewer: ```suggestion
  * Breaking Change: The following deferrals now implement IDisposable: [ScriptDialogOpening](./reference/dotnet/0-9-538/microsoft-web-webview2-core-corewebview2.md#scriptdialogopening), [NewWindowRequested](./reference/dotnet/0-9-538/microsoft-web-webview2-core-corewebview2.md#newwindowrequested), [WebResourceRequested](./reference/dotnet/0-9-538/microsoft-web-webview2-core-corewebview2.md#webresourcerequested), and [PermissionRequested](./reference/dotnet/0-9-538/microsoft-web-webview2-core-corewebview2.md#permissionrequested).
```

# Release notes for WebView2 SDK  

The WebView2 team will be delivering updates to the [WebView2 SDK][WebView2NuGetGallery] on a 6 week cadence. This page will keep you up to date with: product announcements, additions and modifications to the API surface, and breaking changes. 

## 0.X.XXX

[NuGet package][WebView2NuGetGallery0.X.XXX] | minimum Microsoft Edge version XX.X.XXX.X.

**Re-compile your app after updating the NuGet package.**

#### General
* Dropping support for SDK Version [0.8.149](https://docs.microsoft.com/en-us/microsoft-edge/webview2/releasenotes#08149). We recommend staying up to date with the latest version of WebView2.
* Updated group policy to account for when the profile path of Edge is modified  ([#179](https://github.com/MicrosoftEdge/WebViewFeedback/issues/179))

#### Win32 C/C++
* Updated [NewWindowRequested]() to include windowfeatures. Now, ([#70](https://github.com/MicrosoftEdge/WebViewFeedback/issues/70 ))

> * **Breaking Change:** [CreateCoreWebView2EnvironmentWithDetails]() has been deprecated and replaced with [CreateCoreWebView2EnvironmentWithOptions]()
> * **Breaking Change:** In order to ensure that our API aligns with the Windows API naming conventions, we have updated the names of the following interface:

Reviewer 2: ```suggestion
* Breaking Change: [CreateCoreWebView2EnvironmentWithDetails]() has been deprecated and replaced with [CreateCoreWebView2EnvironmentWithOptions]()
* Breaking Change: In order to ensure that our API aligns with the Windows API naming conventions, we have updated the names of the following interface:
```

  * [AreRemoteObjectsAllowed]() is now [AreHostObjectsAllowed]()

> * Updated [AddRemoteObject]() to ensure original host object serializer markers are set to the proxy objects and serialized back as host object when passed as a parameter in the JavaScript callback. ([#148](https://github.com/MicrosoftEdge/WebViewFeedback/issues/148))

Reviewer 2: ```suggestion
* Updated [AddRemoteObject]() to ensure original host object serializer markers are set to the proxy objects and serialized back as a host object when passed as a parameter in the JavaScript callback. ([#148](https://github.com/MicrosoftEdge/WebViewFeedback/issues/148))
```

#### .NET

> * Released WinForms and WPF WebView2API Samples - comprehensive guides of our SDK. Check out the [WebView2 Samples Repo](https://github.com/MicrosoftEdge/WebView2Samples)!

Reviewer 2: ```suggestion
* Released WinForms and WPF WebView2API Samples, which are comprehensive guides of our SDK. Check out the [WebView2 Samples Repo](https://github.com/MicrosoftEdge/WebView2Samples).
```

> * Added support for the following [experimental APIs](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/versioning#experimental-apis): 
> * **Breaking Change:** The following deferrals now implement IDisposable: [ScriptDialogOpening](), [NewWindowRequested](), [WebResourceRequested](), and [PermissionRequested]().
> * Added [GetAvailableBrowserVersionString]() and [CompareBrowserVersions]() as [CoreWebView2Environment]() statics.

Reviewer 2: ```suggestion
* Added support for the following [experimental APIs](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/versioning#experimental-apis): 
  * Breaking Change: The following deferrals now implement IDisposable: [ScriptDialogOpening](), [NewWindowRequested](), [WebResourceRequested](), and [PermissionRequested]().
  * Added [GetAvailableBrowserVersionString]() and [CompareBrowserVersions]() as [CoreWebView2Environment]() statics.
```

# Release notes for WebView2 SDK  

The WebView2 team will be delivering updates to the [WebView2 SDK][WebView2NuGetGallery] on a 6 week cadence. This page will keep you up to date with: product announcements, additions and modifications to the API surface, and breaking changes. 

> [!IMPORTANT]
> Re-compile your app after updating the NuGet package.

## 0.X.XXX

[NuGet package][WebView2NuGetGallery0.X.XXX] | minimum Microsoft Edge version XX.X.XXX.X.

#### General

> * Dropping support for SDK Version [0.8.149](https://docs.microsoft.com/en-us/microsoft-edge/webview2/releasenotes#08149). We recommend staying up to date with the latest version of WebView2.

Reviewer: ```suggestion
* Dropping support for SDK Version [0.8.149](#08149). We recommend staying up to date with the latest version of WebView2.
```

* Updated group policy to account for when the profile path of Edge is modified  ([#179](https://github.com/MicrosoftEdge/WebViewFeedback/issues/179))

#### Win32 C/C++
* Updated [NewWindowRequested]() to include windowfeatures. Now, ([#70](https://github.com/MicrosoftEdge/WebViewFeedback/issues/70 ))
* Breaking Change: [CreateCoreWebView2EnvironmentWithDetails]() has been deprecated and replaced with [CreateCoreWebView2EnvironmentWithOptions]()
* Breaking Change: In order to ensure that our API aligns with the Windows API naming conventions, we have updated the names of the following interface:
  * [AreRemoteObjectsAllowed]() is now [AreHostObjectsAllowed]()

> * Updated [AddRemoteObject]() to ensure original host object serializer markers are set to the proxy objects and serialized back as a host object when passed as a parameter in the JavaScript callback. ([#148](https://github.com/MicrosoftEdge/WebViewFeedback/issues/148))

Reviewer: ```suggestion
* Updated [AddHostObjectToScript](./reference/dotnet/0-9-538/microsoft-web-webview2-core-corewebview2.md#addhostobjecttoscript) to ensure original host object serializer markers are set to the proxy objects and serialized back as a host object when passed as a parameter in the JavaScript callback. ([#148](https://github.com/MicrosoftEdge/WebViewFeedback/issues/148))
```
